js/app.js

```diff
@@ -349,6 +358,33 @@ App.prototype.initSnippetPreview = function() {
 };
 
 /**
+ * Initializes the assessorpresenters for content and SEO.
+ */
+App.prototype.initAssessorPresenters = function() {
+	var targets = {
+		output: this.config.targets.output
+	};
+
+	// Pass the assessor result through to the formatter
+	this.seoAssessorPresenter = new AssessorPresenter( {
+		targets: targets,
+		assessor: this.seoAssessor,
+		i18n: this.i18n
+	} );
+
+	targets = {
```

Comment:
Feels strange overruling the variable used for the SEO Assessor. Why not create a new `var` or directly pass the targets object to the `AssessorPresenter`?